data-reconciliation-app/test/testing.md

```diff
@@ -0,0 +1,76 @@
+# Testing Strategy
```

Comment:
If this file is going to live in the test folder. Do you think it might be better to rename it Readme.md and then GitHub will render it automatically if viewing the code through the browser? Just a thought!

Reply:
Done